CMakeLists.txt

```diff
@@ -17,11 +17,11 @@ set(CLANG_MINIMUM_VERSION "3.4")
 # Also remember to set PROTOCOL_VERSION in network/networkprotocol.h when releasing
 set(VERSION_MAJOR 5)
 set(VERSION_MINOR 1)
-set(VERSION_PATCH 0)
+set(VERSION_PATCH 1)
 set(VERSION_EXTRA "" CACHE STRING "Stuff to append to version string")
 
 # Change to false for releases
-set(DEVELOPMENT_BUILD FALSE)
+set(DEVELOPMENT_BUILD TRUE)
```

Comment:
release commit is still missing

Reply:
This will be done when merging into stable-5

Reply:
Why?

Reply:
because rebasing everytime to move the release commit to the end is annoying

Reply:
`git rebase -i HEAD~10` is only annoying because it touches all files, hence rebuilding is needed.
Not an excuse IMO.

Reply:
it would be `git rebase -i HEAD~2` to reorder each time